api/test-helpers.js

```diff
@@ -17,6 +16,23 @@ const jujulib = require('./client.js');
     connection itself and the WebSocket instance.
 */
 function makeConnection(t, options, callback) {
+  makeConnectionWithResponse(t, options, {}, callback);
+}
+
+/**
+  Create a Juju connection with the given options and provide it to the given
+  callback. When logging in, the simulated server side automatically returns
+  the appropriate response.
+
+  @param {Object} t The test object.
+  @param {Object} options The connect options.
+  @param {Object} loginResponse
```

Comment:
Missing docstring and there are two empty lines.

Reply:
fixed